exercises/ruby-1-a/.meta/config.json

```diff
@@ -0,0 +1,34 @@
+{
+  "solution_files": ["ruby_1_a.rb"],
+  "test_file": "ruby_1_a_test.rb",
+  "tests": [
```

Comment:
I like the new tests entry. The only disadvantage I can think of is that people could forget to update these. So maybe some sort of warning mechanism that can detect if this section is missing a key would be nice later.

Reply:
Yes. Agreed. I could also definitely generate this for Ruby, and/or add a CI test for Ruby for this. I don't know how generalisable that is.

I am wondering if a test generator could output both files automatically.

Reply:
The C# test generator could definitely do this, and I expect most could.